web-server/sgx-wallet-impl/src/schema/actions.rs

```diff
@@ -221,6 +242,8 @@ impl From<UnlockWalletError> for LoadOnfidoCheckResult {
         match err {
             InvalidWalletId => Self::InvalidAuth,
             InvalidAuthPin => Self::InvalidAuth,
+            //FIXME: Do additional error handling to prevent panic.
+            InvalidOtp => unreachable!(), // This variant should never happen as otp is not a feature required for KYC.
```

Comment:
Think we can remove this as I think OTP errors will only occur in Payment transactions.